projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -142,6 +140,9 @@ const BarChart: FC< BarChartProps > = ( {
 					} ) }
 				</AnimatedBarGroup>
 
+				<AnimatedAxis { ...chartOptions.axis.x } />
+				<AnimatedAxis { ...chartOptions.axis.y } />
```

Comment:
Placed the `AnimatedAxis` components after the Bar component to ensure the axes are always visible.